### DISCUSSION TITLE
How do I get access back after generating new personal access token?

### DISCUSSION THREAD

**Discussion by dwlemmer** [community/community#23369] [New to GitHub]
<p>I got an email “Notice for new authentication token formats” about creating a new personal access token, so I followed those instructions.  Now obviously I can’t access my repositories.  How do I get back to where I was?  There was no follow-up in the email or anywhere on how to get back to work.  I’m not trying to become a git expert, I just need access to my repositories.</p>
<p>I would read how to do this myself but I don’t even know what to search for.</p>
<p>If I try a “git fetch” then it pops up a log-in window, but using my email and password doesn’t work.  In fact I got an e-mail about “Basic authentication using a password to Git is deprecated and will soon no longer work.”</p>
<p>Then just get:<br>
Logon failed, use ctrl+c to cancel basic credential prompt.<br>
Username for ‘<a href="https://github.com" rel="noopener nofollow ugc">https://github.com</a>’: ^C</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You will need to use the PAT you created instead of your password when performing Git operations over HTTPS. So when prompted for your password when doing <code>git fetch</code>, enter the PAT.</p>
